### DISCUSSION TITLE
How to increment the value of the column using Update query?

### DISCUSSION THREAD

**Discussion by kev0960** [SeaQL/sea-orm#700] [Q&A] [answered]
Is there a way to construct the increment query something like

```
UPDATE totals 
   SET total = total + 1
WHERE name = 'bill';
```

in sea orm? Only thing I can think of is doing read-update in a same transaction. But is there a way to do that in a single query?

**Comment by tyt2y3** [comment, marked as answer]
```rust
Entity::update_many()
            .col_expr(Column::Total, Expr::col(Column::Total).add(1))
            .filter(Column::Name.eq("Chris"))
            .exec(conn)
            .await
```

**Reply by kev0960** [reply]
Actually, do we have to add sea_query for Expr? How to get Expr from sea orm?

**Reply by billy1624** [reply]
Hey @kev0960, you can import `sea_query::Expr` with `use sea_orm::sea_query`. SeaORM actually re-export SeaQuery on `lib.rs`.

**Reply by kev0960** [reply]
Oh that's cool. Thank you

**Reply by billy1624** [reply]
Welcomed :D
